templates/index/genesis.html

```diff
@@ -1,15 +1,19 @@
 {{ define "genesis" }}
   <div class="row pt-3">
-    <div class="col-lg-5 my-2">{{ template "depositProgress" . }}</div>
+    <div class="col-lg-5 my-2">{{ template "depositProgress" .Data }}</div>
     <div class="col-lg-2 my-2"></div>
-    <div class="col-lg-5 my-2">{{ template "genesisCountdown" . }}</div>
+    <div class="col-lg-5 my-2">{{ template "genesisCountdown" .Data }}</div>
   </div>
   <div style="position:relative" class="card mt-3 index-stats">
     <div class="card-header pt-3">
       <div class="row">
         {{ template "networkStats" }}
       </div>
     </div>
+    <div class="mt-4">
+      <div class="w-100 my-2" id="slotsViz"></div>
+    </div>
```

Comment:
Doesn't fit within the boundaries.
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/928e78a5-c433-468e-a794-3ad21e782625)
You should rather replace the div with id `checklist` with the slot viz, as this was also where it was inserted before this PR.
```html
<div class="card-body">
  <div class="row">
    <div class="col-lg-12">
      <div class="m-2" id="slotsViz"></div>
      <h5 class="my-3">Genesis Checklist</h5>
```

Reply:
I think it shouldn't be replaced, but rather added alongside. Otherwise we'd also have to adjust the selector [here](https://github.com/gobitfly/eth2-beaconchain-explorer/blob/master/handlers/index.go#L91), which makes it inconsistent because for post genesis there is no checklist, thus that naming would be confusing.

I missed the boundaries issue because on a local testnet the numbers aren't that big, thanks for pointing it out :+1: 

Pushed a new solution, I hope that's fine, lmk what you think :slightly_smiling_face:

Reply:
I don't get why the selector would need to be adjusted. Your new changes look good, but the `checklist` div has no use anymore, right? I believe it can be removed, that's what I meant in the first place.